### PR TITLE
VideoSW: fix some warnings

### PR DESCRIPTION
**Pull request by Tilka**

### CODE REVIEW

File: Source/Core/VideoBackends/Software/SWVertexLoader.h [hunk 1]
```diff
@@ -26,7 +26,7 @@ class SWVertexLoader : public VertexManagerBase
 	NativeVertexFormat* CreateNativeVertexFormat(const PortableVertexDeclaration& vdec) override;
 
 protected:
-	virtual void ResetBuffer(u32 stride);
+	virtual void ResetBuffer(u32 stride) override;
```
Comment (JosJuice) [inline Source/Core/VideoBackends/Software/SWVertexLoader.h]: Remove `virtual`. `override` makes it unnecessary.

**Comment by degasus** [comment]
My bad, seems like I've missed them. LGTM but the unneeded virtual.

File: Source/Core/VideoBackends/Software/SWmain.cpp [hunk 2]
```diff
@@ -73,7 +64,7 @@ class PerfQuery : public PerfQueryBase
 		return EfbInterface::perf_values[type];
 	};
 	void FlushResults() override {}
-	bool IsFlushed() const {return true;};
+	bool IsFlushed() const override { return true; };
```
Comment (delroth) [inline Source/Core/VideoBackends/Software/SWmain.cpp]: The final ; on that line is unnecessary.

**Comment by delroth** [comment]
@dolphin-emu-bot allowmerge

**Comment by Tilka** [comment]
Ugh, sorry, I scrolled up too late.

**Pull request merged**